extractors/python_functions/quote_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "quote"
 
-def quote_extraction(record) -> List[Tuple[str, int, int]]:
+def quote_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.